frontend/src/components/Header/Header.jsx

```diff
@@ -79,13 +81,15 @@ const Header = () => {
               <span>Term Planner</span>
               <Link to="/term-planner" />
             </Menu.Item>
-            {/* <Menu.Item key="progression-checker">
+            { inDev && (
+            <Menu.Item key="progression-checker">
               <span>Progression Checker</span>
-              <Link to="/progression-checker"></Link>
-            </Menu.Item> */}
+              <Link to="/progression-checker" />
+            </Menu.Item>
+            )}
           </Menu>
           <PlannerCart />
-          {/* <ThemeToggle /> */}
+          { inDev && <ThemeToggle /> }
```

Comment:
small styling thing but inconsistent spacing in braces compared to some other files like drawercontent